ime/app/src/main/java/com/anysoftkeyboard/AnySoftKeyboard.java

```diff
@@ -667,15 +685,29 @@ private void onNonFunctionKey(
             default:
                 if (isWordSeparator(primaryCode)) {
                     handleSeparator(primaryCode);
-                } else {
-                    if (mControlKeyState.isActive() && primaryCode >= 32 && primaryCode < 127) {
+                } else if (mControlKeyState.isActive()) {
+                    int keyCode = getKeyCode(primaryCode);
+                    if (BuildConfig.DEBUG)
```

Comment:
No need for these logs. Please remove as many loggings as makes sense

Reply:
Done